webbpsf/gridded_library.py

```diff
@@ -329,10 +341,12 @@ def create_grid(self):
             meta["DIFFLMT"] = (psf[ext].header["DIFFLMT"], "Diffraction limit lambda/D in arcsec")
             meta["FFTTYPE"] = (psf[ext].header["FFTTYPE"], "Algorithm for FFTs: numpy or fftw")
             meta["NORMALIZ"] = (psf[ext].header["NORMALIZ"], "PSF normalization method")
-            meta["JITRTYPE"] = (psf[ext].header["JITRTYPE"], "Type of jitter applied")
-            meta["JITRSIGM"] = (psf[ext].header["JITRSIGM"], "Gaussian sigma for jitter [arcsec]")
             meta["TEL_WFE"] = (psf[ext].header["TEL_WFE"], "[nm] Telescope pupil RMS wavefront error")
 
+            if self.webb.name != "WFI":
```

Comment:
Hmm, I'm not a fan of having these keywords be instrument-dependent. It seems like it would be more consistent overall to write these keywords for WFI too, but with values like `None` or `0` to indicate that such models are not applied. Better yet, we should add in the actual convolution step for WFI eventually. (This is out of scope for this PR, just want to note it's something we should do eventually) . Do you agree?

Reply:
Right,`WFI` PSFs don't have `JITRTYPE` or `JITRSIGM` keywords, so they can't be carried through here from the PSF header to the `GriddedPSFModel` object. I can set these values to 0 for the `GriddedPSFModel` object here, but they still wouldn't exist for a `WFI` PSF, since that change would have to come from outside this grid method. At this point (knowing it'll be fixed outside this special case in the future), do you think it's better for the `GriddedPSFModel` header to match the PSF header for `WFI` or to have the model's all have the same headers but have WFI model not match it's PSF?

Reply:
Oh I'm just going to modify the WFI class so that every WFI PSF also has the same keywords for jitter. I didn't realize they were inconsistent or we would have just fixed that long ago so they all match.